recipes/libnetcdf/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "4.4.0" %}
+
+package:
+    name: libnetcdf
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/Unidata/netcdf-c.git
+    git_tag: v{{ version }}
+
+build:
+    number: 0
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+
+requirements:
+    build:
+        - m4
+        - curl
+        - hdf5
+        - zlib
+    run:
+        - curl
+        - hdf5
+        - zlib
+
+test:
+    commands:
+        - ncdump  # [unix]
+        - nc-config --help  # [unix]
+
+about:
+    home: http://www.unidata.ucar.edu/software/netcdf/
+    license: MIT
+    summary: libraries and data formats that support array-oriented scientific data
+
+extra:
+    recipe-maintainers:
+        - jakirkham
+        - rgrout
```

Comment:
@jakirkham and @groutr I added you here because of `hdf5`. Let me know if that is OK for you.